sdk/portal/src/utils/wait-for-transaction-receipt.ts

```diff
@@ -24,16 +41,10 @@ import { type WebsocketClientOptions, getWebsocketClient } from "./websocket-cli
  * @property {boolean} isContract - Whether the transaction is a contract deployment
  */
 export interface Transaction {
-  receipt: {
-    transactionHash: string;
-    to: string;
-    status: string;
-    from: string;
-    type: string;
+  receipt: TransactionReceipt & {
```

Comment:
Should be 'TransactionReceipt<string>', bigints are converted to a string

Reply:
Also the 'status' property' in graphql is different. It changes the casing because we use an enum.

Should be `status: 'Success' | 'Reverted'`

Reply:
Ah ok! Because I was getting a type error when I imported the type from the file. I'll do a .toLowerCase and then compare so that both are handled.